packages/hyperion-autologging/src/ALUIEventPublisher.ts

```diff
@@ -176,7 +201,12 @@ export function publish(options: InitOptions): void {
   let lastUIEvent: CurrentUIEvent | null;
 
   uiEvents.forEach((eventConfig => {
-    const { eventName, cacheElementReactInfo = false } = eventConfig;
+    const { eventName } = eventConfig;
+    if (eventName === 'hover') {
+      ALHoverPublisher.publish({ channel, uiEvents, hoverDurationThresholdMS: eventConfig.hoverDurationThresholdMS ?? 0 });
```

Comment:
I don't particularly like this,  and I saw what we do with ElementValue publisher but this one is a bit different in that `hover` is not a legitimate eventName from `DocumentEventMap` to reuse so this causes a bit of type problems with emitting `al_ui_event`s and also reusing the typing in `UIEventConfig`.

But I'm open to attempting to make it work in that way,  or open to other thoughts.

We may consider we have other `derived` UI events in the future,  and maybe can come up with what's the best way to support them in initialization and on typing the events on the channel.

Reply:
cc: @vsylee for thoughts on triggerFlowlet integration

Reply:
we we discussed today, if you retire uiEventFlowlet, then you may not need to worry about flowlet.
In anycase, we should not special case flowlets for hover or any other ui event.